repo.json

```diff
@@ -1,12 +1,2583 @@
 {
-    "repo_name": "O3DE Extras - Extras gems, projects, and templates for O3DE",
+    "repo_name": "O3DE Extras",
     "origin": "o3de-extras",
     "repo_uri": "https://raw.githubusercontent.com/o3de/o3de-extras/development",
     "summary": "A repository to hold various gems, projects, and templates for the O3DE engine.",
-    "additional_info": "See the README.md at the root of this repository for more information",
+    "additional_info": "Extras gems, projects, and templates for O3DE. See the README.md at the root of this repository for more information",
     "last_updated": "2022-09-27",
+    "$schemaVersion": "1.0.0",
+    "gems_data": [
+        {
+            "gem_name": "AzQtComponentsForPython",
+            "display_name": "AzQtComponents For Python",
+            "license": "Apache-2.0 Or MIT",
+            "license_url": "https://github.com/o3de/o3de/blob/development/LICENSE.txt",
+            "origin": "Open 3D Engine - o3de.org",
+            "origin_url": "https://github.com/o3de/o3de",
+            "type": "Code",
+            "summary": "This gem provides a library to use the AzQtComponents library from Python.",
+            "canonical_tags": [
+                "Gem"
+            ],
+            "user_tags": [
+                "Scripting",
+                "UI"
+            ],
+            "icon_path": "preview.png",
+            "requirements": "",
+            "documentation_url": "",
+            "dependencies": [
+                "EditorPythonBindings"
+            ],
+            "repo_uri": "https://raw.githubusercontent.com/o3de/o3de-extras/development",
+            "version": "1.0.0",
+            "versions_data": [
+                {
+                    "download_source_uri": "https://github.com/o3de/o3de-extras/releases/download/2.0/azqtcomponentsforpython-1.0.0-gem.zip",
+                    "sha256": "ef75e9811b11e081bd4d16d62b638208fe9f0bd8966cfaff937e64b59343f5f7",
+                    "version": "1.0.0"
+                }
+            ]
+        },
```

Comment:
Was this repo.json file generated?

Because if not it seems cumbersome to have to add in a copy of each gems.json and template.json data into this file.

Reply:
I like the idea of a dedicated GitHub Action to do this for us though as we push changes.